crm_won_reason/__init__.py

```diff
@@ -1,2 +1,27 @@
 from . import models
 from . import wizard
+from odoo import api, SUPERUSER_ID
+from logging import getLogger
+
+_logger = getLogger(__name__)
+
+
+def pre_init_hook(cr):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    # in databases where sale and crm are both installed sale_crm is auto installed
+    # sale_crm declares view crm_case_form_view_oppor which looks for
+    # <xpath expr="//button[@name='action_set_won_rainbowman']"
+    # but crm_won_reason changes the name of action_set_won_rainbowman
+    # so "button cannot be located in parent view" error is raised
+    # crm_won_reason depends only on crm, so we need to remove sale_crm
```

Comment:
@ivantodorovich this is the issue that happens on this PR. And I thought it can be solve such way. But, nope `sale_crm` is going to be installed after this one.
Do you have any ideas?

Reply:
>  but crm_won_reason changes the name of action_set_won_rainbowman

why?

Reply:
because we need to use new action name
```
<xpath
    expr="//button[@name='action_set_won_rainbowman']"
    position="attributes"
>
    <attribute
        name='name'
    >%(crm_won_reason.crm_lead_win_action)d</attribute>
    <attribute name='type'>action</attribute>
</xpath>
```
so name isn't `action_set_won_rainbowman` anymore after `crm_won_reason` is installed

Reply:
Then hide the original one and yours. Isn't this an option?

Reply:
@simahawk done it and checked on the project, seems good, thanks for the review